### PR TITLE
filter results based on user date input

### PR DESCRIPTION
**Pull request by cmdctrlesc**

### CODE REVIEW

File: back/views/default.py [hunk 1]
```diff
@@ -139,14 +139,19 @@ def remove_marker(request):
 
 @view_config(route_name='list_markers', renderer='json')
 def list_markers(request):
+    #min_date = ...
```
Comment (JanLikar) [inline back/views/default.py]: Just obtain it from GET parameters.

I think it's something like `request.params.get('min_date')`

Comment (JanLikar) [reply back/views/default.py]: You can test it by going to the "/list-markers?min_date=2020-03-03"

**Review by JanLikar — CHANGES_REQUESTED** [review]
Looks good :+1:

File: back/views/default.py (hunk 1 above)
Comment (cmdctrlesc) [reply back/views/default.py]: Thanks, I did that.

**Review by JanLikar — APPROVED** [review]

**Pull request merged**